runtime/v2/vm/src/main/java/com/walmartlabs/concord/svm/InMemoryState.java

```diff
@@ -216,6 +217,59 @@ public Exception clearThreadError(ThreadId threadId) {
         }
     }
 
+    @Override
+    public List<StackTraceItem> getStackTrace(ThreadId threadId) {
+        synchronized (this) {
```

Comment:
Change to
```java
public synchronized List<StackTraceItem> getStackTrace(ThreadId threadId)
```
?

`synchronized (this)` that spans the method's whole body is essentially just a synchronized method, AFAIR?

Reply:
yeah. same. but as you can see in this class all methods uses 
```
synchronized (this)
```
decided not to change the style :)

Reply:
Yeah, not worth it.